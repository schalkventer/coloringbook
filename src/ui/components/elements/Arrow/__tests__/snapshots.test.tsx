import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { Default, Change, Directions, Minimal } from './stories';

// ⚙️ Elements
// 🔨 Custom
// 📺 Views

describe('UI Components', (): void => {
  describe('Experimental', (): void => {
    describe('Arrow', (): void => {
      describe('snapshots', (): void => {
        test('Default', (): void => {
          expect(renderer.create(<Default />).toJSON()).toMatchSnapshot();
        });

        test('Change', (): void => {
          expect(renderer.create(<Change />).toJSON()).toMatchSnapshot();
        });

        test('Directions', (): void => {
          expect(renderer.create(<Directions />).toJSON()).toMatchSnapshot();
        });

        test('Minimal', (): void => {
          expect(renderer.create(<Minimal />).toJSON()).toMatchSnapshot();
        });
      });
    });
  });
});
