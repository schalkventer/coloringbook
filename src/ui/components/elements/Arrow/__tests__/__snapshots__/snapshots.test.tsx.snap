// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI Components Experimental Arrow snapshots Change 1`] = `
Array [
  <div>
    <button
      onClick={[Function]}
      type="button"
    >
      Change
    </button>
  </div>,
  .c0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
}

.c1 {
  opacity: 0.2;
  width: 13px;
  height: 13px;
}

<div
    className="c0"
    style={
      Object {
        "transform": "rotate(0deg)",
      }
    }
  >
    <svg
      aria-hidden="true"
      className="MuiSvgIcon-root c1"
      focusable="false"
      role="presentation"
      viewBox="0 0 24 24"
    >
      <path
        d="M0 0h24v24H0z"
        fill="none"
      />
      <path
        d="M12 4l-1.41 1.41L16.17 11H4v2h12.17l-5.58 5.59L12 20l8-8z"
      />
    </svg>
  </div>,
  .c0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
}

.c1 {
  opacity: 0.2;
  width: 13px;
  height: 13px;
}

<div
    className="c0"
    style={
      Object {
        "transform": "rotate(0deg)",
      }
    }
  >
    <svg
      aria-hidden="true"
      className="MuiSvgIcon-root c1"
      focusable="false"
      role="presentation"
      viewBox="0 0 24 24"
    >
      <path
        d="M0 0h24v24H0z"
        fill="none"
      />
      <path
        d="M12 4l-1.41 1.41L16.17 11H4v2h12.17l-5.58 5.59L12 20l8-8z"
      />
    </svg>
  </div>,
]
`;

exports[`UI Components Experimental Arrow snapshots Default 1`] = `
.c0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
}

.c1 {
  opacity: 0.2;
  width: 13px;
  height: 13px;
}

<div
  className="c0"
  style={
    Object {
      "transform": "rotate(0deg)",
    }
  }
>
  <svg
    aria-hidden="true"
    className="MuiSvgIcon-root c1"
    focusable="false"
    role="presentation"
    viewBox="0 0 24 24"
  >
    <path
      d="M0 0h24v24H0z"
      fill="none"
    />
    <path
      d="M12 4l-1.41 1.41L16.17 11H4v2h12.17l-5.58 5.59L12 20l8-8z"
    />
  </svg>
</div>
`;

exports[`UI Components Experimental Arrow snapshots Directions 1`] = `
Array [
  .c0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
}

.c1 {
  opacity: 0.2;
  width: 13px;
  height: 13px;
}

<div
    className="c0"
    style={
      Object {
        "transform": "rotate(0deg)",
      }
    }
  >
    <svg
      aria-hidden="true"
      className="MuiSvgIcon-root c1"
      focusable="false"
      role="presentation"
      viewBox="0 0 24 24"
    >
      <path
        d="M0 0h24v24H0z"
        fill="none"
      />
      <path
        d="M12 4l-1.41 1.41L16.17 11H4v2h12.17l-5.58 5.59L12 20l8-8z"
      />
    </svg>
  </div>,
  .c0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
}

.c1 {
  opacity: 0.2;
  width: 13px;
  height: 13px;
}

<div
    className="c0"
    style={
      Object {
        "transform": "rotate(90deg)",
      }
    }
  >
    <svg
      aria-hidden="true"
      className="MuiSvgIcon-root c1"
      focusable="false"
      role="presentation"
      viewBox="0 0 24 24"
    >
      <path
        d="M0 0h24v24H0z"
        fill="none"
      />
      <path
        d="M12 4l-1.41 1.41L16.17 11H4v2h12.17l-5.58 5.59L12 20l8-8z"
      />
    </svg>
  </div>,
  .c0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
}

.c1 {
  opacity: 0.2;
  width: 13px;
  height: 13px;
}

<div
    className="c0"
    style={
      Object {
        "transform": "rotate(180deg)",
      }
    }
  >
    <svg
      aria-hidden="true"
      className="MuiSvgIcon-root c1"
      focusable="false"
      role="presentation"
      viewBox="0 0 24 24"
    >
      <path
        d="M0 0h24v24H0z"
        fill="none"
      />
      <path
        d="M12 4l-1.41 1.41L16.17 11H4v2h12.17l-5.58 5.59L12 20l8-8z"
      />
    </svg>
  </div>,
  .c0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
}

.c1 {
  opacity: 0.2;
  width: 13px;
  height: 13px;
}

<div
    className="c0"
    style={
      Object {
        "transform": "rotate(270deg)",
      }
    }
  >
    <svg
      aria-hidden="true"
      className="MuiSvgIcon-root c1"
      focusable="false"
      role="presentation"
      viewBox="0 0 24 24"
    >
      <path
        d="M0 0h24v24H0z"
        fill="none"
      />
      <path
        d="M12 4l-1.41 1.41L16.17 11H4v2h12.17l-5.58 5.59L12 20l8-8z"
      />
    </svg>
  </div>,
]
`;

exports[`UI Components Experimental Arrow snapshots Minimal 1`] = `
.c0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
}

.c1 {
  opacity: 0.2;
  width: 13px;
  height: 13px;
}

<div
  className="c0"
  style={
    Object {
      "transform": "rotate(0deg)",
    }
  }
>
  <svg
    aria-hidden="true"
    className="MuiSvgIcon-root c1"
    focusable="false"
    role="presentation"
    viewBox="0 0 24 24"
  >
    <path
      d="M5.88 4.12L13.76 12l-7.88 7.88L8 22l10-10L8 2z"
    />
    <path
      d="M0 0h24v24H0z"
      fill="none"
    />
  </svg>
</div>
`;
